tfhe/src/integer/gpu/server_key/radix/mod.rs

```diff
@@ -147,6 +147,15 @@ impl CudaServerKey {
         res
     }
 
+    pub(crate) fn encoding(&self, padding_bit: PaddingBit) -> ShortintEncoding {
+        ShortintEncoding {
+            ciphertext_modulus: self.ciphertext_modulus,
+            message_modulus: self.message_modulus,
+            carry_modulus: self.carry_modulus,
+            padding_bit,
```

Comment:
Shouldn't we hardcode `PaddingBit::Yes` here
Or otherwise store it in `CudaServerKey`